### DISCUSSION TITLE
Ring MQTT not taking account credentials

### DISCUSSION THREAD

**Discussion by 4rdtough** [tsightler/ring-mqtt#941] [Support Q&A]
When I try to log in to ring MQTT it doesn't show anything after I press submit, and there are no errors or anything. I've only had home assistant for a year and am fairly new so I could be doing it wrong

**Comment by tsightler** [comment]
There is a bug in the current web authenticator app that prevents the error message from being displayed correctly in the web UI itself, however, after clicking submit, if nothing happens, you can quickly switch to the addon logs view (or keep the logs view open in a separate tab) and the actual returned error will display in the log in red.
